app/classifier/tasks/survey/choice.spec.js

```diff
@@ -29,17 +29,28 @@ describe('Choice', function () {
   let wrapper;
   let answer;
 
-  describe('with single answer questions', function () {
-    beforeEach(function () {
-      wrapper = shallow(<Choice
-        translation={task}
-        task={task}
-        annotation={annotation}
-        annotationValue={annotationValue}
-        choiceID='ar'
-      />);
-      answer = wrapper.find('input[name="ho"][value="two"]');
+  beforeEach(function () {
+    wrapper = shallow(<Choice
+      translation={task}
+      task={task}
+      annotation={annotation}
+      annotationValue={annotationValue}
+      choiceID='ar'
+    />);
+    answer = wrapper.find('input[name="ho"][value="two"]');
+  });
+
+  describe('with confused with options', function () {
+    it('should render the confusions with appropriate labels', function () {
+      const confusions = wrapper.find('.survey-task-confusions-modal');
+      assert.equal(confusions.length, task.choices['ar'].confusionsOrder.length);
+      confusions.forEach(function (confusion, i) {
+        assert.equal(confusion.dive().find('.survey-task-choice-confusion').text(), task.choices[task.choices['ar'].confusionsOrder[i]].label);
```

Comment:
I think `confusion` is an enzyme wrapper around a modal form component here, so you could test `confusion.prop('trigger').text()` to check the label. It's not worth spending a lot of time on, though.

Reply:
I've updated these tests to remove the CSS class selectors. For this test, I just test the React component that's passed to `trigger` without really caring how it's rendered.

Reply:
This looks much better, thanks!